src/Microsoft.AspNetCore.Hosting/Startup/ExceptionPage/Views/ErrorPage.cshtml

```diff
@@ -156,5 +156,8 @@
             <%$ include: ErrorPage.js %>
             //-->
         </script>
-    </body>
+        <footer>
```

Comment:
@Eilon  Footer here is after the `<script>` tag because our razor tool generator modifies the file in a way which generates incorrect .cs file (the escaping of the double quotes get messed up...snapshot below)...probably should file a separate issue for it?

![image](https://cloud.githubusercontent.com/assets/6559784/17952045/dd45a650-6a1b-11e6-911d-f2ee5f05a0de.png)